### PR TITLE
feat: Implementation of cross_entropy loss in losses.py in ivy/experimental…

### LINKED ISSUE #26642: cross_entropy

### PR DESCRIPTION
**Pull request by Huvinesh-Rajendran-12**
….Implementation of cross_entropy loss in each of the following backends: jax,numpy,paddle,tensorflow,torch. Implemented cross_entropy in ivy Array and Container.

# PR Description
This PR is opened to implement the cross_entropy loss to extend the Ivy functional API. 

## Related Issue

Closes #26642 

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### Socials

https://twitter.com/Huvin99

### CODE REVIEW

**Comment by aparajith21** [comment]
Hi! Thanks for contributing! Please add tests appropriately, and resolve merge conflicts,  and request review once that's done
